pytest_fixtures/core/sat_cap_factory.py

```diff
@@ -277,7 +277,7 @@ def module_sat_ready_rhels(request):
 
 @pytest.fixture
 def cap_ready_rhel():
-    rhel_version = Version(settings.capsule.version.release)
+    rhel_version = Version(settings.capsule.version.rhel_version)
```

Comment:
This change should also fix failures in automation

Reply:
Thanks for fixing this:)